internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -456,6 +459,35 @@ func askMongoShellAndSetConfig() error {
 	return nil
 }
 
+func askAtlasAccountAndProfile() error {
+	fmt.Println("missing default profile.")
```

Comment:
[nit] let's output to stderr and change the copy given we changed the logic to check this
```suggestion
	_ = fmt.Printf(os.Stderr, "No API credentials set.\n")
```
(I don't remember if this is the correct syntax for staderr)